app/models/fagus/item.rb

```diff
@@ -0,0 +1,35 @@
+# Fagus
+module Fagus
+  # Represents an item.
+  class Item < ActiveRecord::Base
+    extend Enumerize
+
+    belongs_to :category
+    belongs_to :item_type
+
+    enumerize :status,
+              :in => [:active, :phased_out, :discontinued],
+              :default => :active
+
+    validates \
+      :category,
+      :description,
+      :item_type,
+      :reference,
+      :status,
```

Comment:
I just tried it... it looks like enumerize won't set the value if its not one of the valid values.

```
2.2.1 :001 > item = Fagus::Item.new
 => #<Fagus::Item id: nil, category_id: nil, description: nil, item_type: nil, reference: nil, status: "active", uuid: nil, deleted_at: nil, created_at: nil, updated_at: nil>
2.2.1 :002 > item.status = :not_valid
 => :not_valid
2.2.1 :003 > item.status
 => nil
2.2.1 :004 > item.status = :active
 => :active
2.2.1 :005 > item.status
 => "active"
```